admin/base/settings/defaults.py

```diff
@@ -219,3 +224,10 @@
 
 ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
                 'institution_campaign': 'institution'}
+
+
+# Change desk settings in local.py
+DESK_KEY = ''
+DESK_KEY_SECRET = ''
+DESK_TOKEN = ''
```

Comment:
I know Desk allows this token and `DESK_TOKEN_SECRET` to be manually generated, and used to authenticate on behalf of a Desk user. 

Should these be attached to a user, rather than stored in the settings? This would allow us to have multiple Desk users managing cases, and wouldn't be a horrible security risk because these are useless without the `KEY` and `KEY_SECRET`